### ISSUE TITLE
msf/msf-inject fails to send payload

### ISSUE THREAD

**Issue by dhauenstein** [BishopFox/sliver#581]
**Describe the bug**
The `msf` and `msf-inject` commands always fail with an error:

```
sliver (ORIGINAL_COMPLAINT) > msf --lhost 10.10.11.3 --lport 8443 

[!] rpc error: code = InvalidArgument desc = Missing session request field
```

**To Reproduce**
1. Open session in sliver with active Windows host
2. Execute `msf --lhost 10.10.11.3 --lport 8443`
3. See error

**Expected behavior**
The metasploit payload should be generated and send to the host, and a meterpreter session should be established. 

**Desktop (please complete the following information):**
 - OS: Kali Linux
 - Version 1.5.2
 - Metersploit Version: 6.1.14-dev

**Comment by moloch--** [comment]
Likely a type error due to session id's now being uuids

**Comment by moloch--** [comment]
Fixed in v1.5.3
